packages/embarkjs/src/utils.js

```diff
@@ -1,6 +1,12 @@
 /* global Web3 clearInterval setInterval */
 
 let Utils = {
+  hexPrefix: function(str) {
+    if (str === undefined) return;
```

Comment:
```suggestion
    if (str && !str.match) return;
```

Reply:
`str === undefined` will not early return for `null`, `true`, etc. It's not a big deal but the suggested changed makes the early return test a bit more comprehensive.